service/history/queue/timer_queue_processor.go

```diff
@@ -215,6 +216,15 @@ func (t *timerQueueProcessor) Stop() {
 	for _, standbyTaskExecutor := range t.standbyTaskExecutors {
 		standbyTaskExecutor.Stop()
 	}
+
+	if len(t.failoverQueueProcessors) > 0 {
+		t.logger.Info("Shutting down failover timer queues", tag.Counter(len(t.failoverQueueProcessors)))
+		for _, failoverQueueProcessor := range t.failoverQueueProcessors {
+			failoverQueueProcessor.Stop()
+		}
+	}
+
+	t.activeTaskExecutor.Stop()
```

Comment:
why is the order changed?

Reply:
`t.activeTaskExecutor` is used by failover queues so stopping it after failover queues